pep-9999.rst

```diff
@@ -35,21 +35,24 @@ stubs serve multiple purposes:
   members.
 
 This PEP aims to give guidance to both authors of type stubs and developers
-of type checkers. It describes the contents of compliant type stubs,
-suggests a style guide for them, and lists constructs that compliant type
+of type checkers. It describes the constructs that can be used safely in type stubs,
+suggests a style guide for them, and lists constructs that type
 checkers are expected to support.
 
-A type stub is considered to be compliant if it only uses constructs decribed
-in this PEP. Type stub authors can elect to use additional constructs, but
+Type stubs that only use constructs described in this PEP should work with
+all type checkers that also follow this PEP.
+Type stub authors can elect to use additional constructs, but
 must be prepared that some type checkers will not parse them as expected.
 
-A type checker is considered to be compliant if
-it accepts all compliant type stubs. This means it
-will parse a compliant type stub without error and does not interpret any
-construct in a contradictory manner. A compliant type checker is not
-required to interpret all standard constructs, and additionally a type checker
-can support extra constructs not described in this PEP. All deviations from
-this PEP should be documented.
+A type checker that conforms to this PEP will parse a type stub only using
+constructs described here without error and does not interpret any
+construct in a contradictory manner. Type checkers are not
+required to implement checks for all these constructs, though, but
+can elect to ignore unsupported ones. Additionally a type checker
```

Comment:
nit: I find the first two sentences of this paragraph a little clunky. I'd suggest rewording them as:

A type checker that conforms to this PEP will parse a type stub ~only using~ *that only uses* constructs described here without error and ~does~ *will* not interpret any construct in a contradictory manner. *However,* type checkers are not required to implement checks for all these constructs ~, though, but~ *and* can elect to ignore unsupported ones.